sections/credits.tex

```diff
@@ -0,0 +1,32 @@
+% !TeX spellcheck = en_US
+
+\addsection{Credits}{\images/experience.png}
+
+\iftoggle{printable}{\vspace{-\baselineskip}}{}
+
+\bigbreak
+
+\begin{multicols*}{2}
+
+This project shamelessly borrows best practices from the \href{https://github.com/Heegu-sama/Homm3BG}{\textbf{Rule Book Rewrite Project}}, which include LaTeX code, GitHub engineering, and translation framework.
+You should check it out if you haven't seen it yet.
+
+\textbf{Scenario templating and LaTeX writing}: Tomáš Zeman, Andrzej Wiącek, Saveliy Ivanov
```

Comment:
You should put yourself to be the first :)

Reply:
Nah, so far I haven't added even a single scenario 😂

Reply:
But you're the "father" of the project :D

Reply:
Well, time for the godfather to shine!